forge/db/controllers/ProjectTemplate.js

```diff
@@ -64,9 +64,22 @@ module.exports = {
             }
             settings.env.forEach((envVar) => {
                 if (templateEnvPolicyMap[envVar.name] !== false && !/ /.test(envVar.name)) {
+                    if (!envVar.name.match(/^[a-zA-Z_]+[a-zA-Z0-9_]*$/)) {
+                        throw new Error(`Invalid Env Var name ${envVar.name}`)
```

Comment:
```suggestion
                        throw new Error(`Invalid Env Var name '${envVar.name}'`)
```

When a blank name is used, it shows error as 

```
{
  "code": "env_var_validation",
  "error": "Invalid Env Var name "
}
```

Adding single quotes will show error as 

```
{
  "code": "env_var_validation",
  "error": "Invalid Env Var name ''"
}
```